From cc4f64d8930553f0b84ec613f626f5317df303ab Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Wed, 6 Apr 2011 21:44:42 -0500
Subject: Don't clear FONTCONFIG_LIBS and FONTCONFIG_CFLAGS in configure.

The build should no longer fail when linking with --no-add-needed

Previously, configure would clear FONTCONFIG_LIBS and
FONTCONFIG_CFLAGS.  This would break the build when compiling with a
linker that defaults to --no-add-needed.  For example:

  /usr/bin/ld: ftfont.o: undefined reference to symbol 'FcInit'

configure should no longer clear these variables.

Author: Matthias Klose <doko@ubuntu.com>
Date: Mon, 28 Feb 2011 01:02:41 +0100
Added-by: Rob Browning <rlb@defaultvalue.org>
Closes: #554324
---
 configure.in |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/configure.in b/configure.in
index baacfe4..0639715 100644
--- a/configure.in
+++ b/configure.in
@@ -2044,11 +2044,7 @@ if test "${HAVE_X11}" = "yes"; then
     ## We used to allow building with FreeType and without Xft.
     ## However, the ftx font backend driver is not in good shape.
     if test "${HAVE_XFT}" = "yes"; then
-      dnl As we use Xft, we anyway use freetype.
-      dnl There's no need for additional CFLAGS and LIBS.
       HAVE_FREETYPE=yes
-      FONTCONFIG_CFLAGS=
-      FONTCONFIG_LIBS=
     fi
 
     HAVE_LIBOTF=no
-- 
1.7.6.3

